src/actors/common/augeas/actor.py

```diff
@@ -27,7 +29,7 @@ def get_lenses():
         3) Excluded paths
            Currently unused
     '''
-    augeas = subprocess.check_output(["augtool", "--include=lenses", "dump-xml", "/augeas"])
+    augeas = subprocess.check_output(["augtool"] + magic.do_magic() + ["--include=lenses", "dump-xml", "/augeas"])
```

Comment:
Can we go for something like `AugeasHelper.build()`? At first I thought this has something to do with `libmagic`.